Rakefile

```diff
@@ -76,13 +80,13 @@ namespace 'test' do # rubocop:disable Metrics/BlockLength
   end
 end
 
-begin
-  require 'rubocop/rake_task'
-  if RUBY_VERSION >= '2.2.0' && (defined?(RUBY_ENGINE) && RUBY_ENGINE == 'ruby') && ENV['MOCHA_RUN_INTEGRATION_TESTS'].nil?
+task 'lint' do
+  if defined?(RuboCop::RakeTask)
```

Comment:
Since `lint` is a separate job now, we don't need to check for MOCHA_RUN_INTEGRATION_TESTS. I'm not super familiar with RUBY_ENGINE though

Reply:
> Since `lint` is a separate job now, we don't need to check for MOCHA_RUN_INTEGRATION_TESTS. 

Agreed.

> I'm not super familiar with RUBY_ENGINE though

This was there to cope with the JRuby build job. However, now that linting is a separate job we don't need to do these checks any more.